src/webview/js/js.js

```diff
@@ -740,9 +740,22 @@ const handleLongPress = (target: Element) => {
 
   hasLongPressed = true;
 
+  if (target.closest('.reaction-list')) {
+    const reactionNode = target.closest('.reaction');
+    if (!reactionNode) {
+      return;
+    }
+    sendMessage({
+      type: 'reactionDetails',
+      messageId: getMessageIdFromNode(target),
+      targetReactionName: reactionNode.getAttribute('data-name') || '',
```

Comment:
As we discussed by voice today: let's use `requireAttribute`, like the other similar code around this does.